tests/e2e/CMakeLists.txt

```diff
@@ -0,0 +1,226 @@
+
+############################################# Add E2E Tests ################################################
```

Comment:
ms banner? and min version check?

Reply:
Ah yup this is left over when e2e and ut were combined in one. For the min version check though that only needs to be before the first `project()` call which is in the top level. It defines the min version for the whole project there.